templates/contribute/naming.md

```diff
@@ -259,6 +259,40 @@ check le_of_mul_le_mul_left
 check le_of_mul_le_mul_right
 ```
 
+## Naming of structural lemmas ##
+
+We are trying to standardize certain naming patterns for structural lemmas.
+At present these are not uniform across mathlib.
+
+### Extensionality ###
+
+A lemma of the form `(∀ a, a ∈ s₁ ↔ a ∈ s₂) → s₁ = s₂` should be named `.ext`,
+and labelled with the `@[ext]` attribute.
+Often this type of lemma can be generated automatically by putting the
+`@[ext]` attribute on a structure.
```

Comment:
AFAIK `@[ext]` uses projections while quite often `ext` lemmas should use coercions.

Reply:
Good point. I will make this explicit.